### DISCUSSION TITLE
Please support dynamically loading "Single File Components" when Vue is loaded from a CDN

### DISCUSSION THREAD

**Discussion by boustanihani** [vuejs/rfcs#684] [General]
According to: https://vuejs.org/guide/quick-start.html#using-vue-from-cdn
`When using Vue from a CDN, there is no "build step" involved. This makes the setup a lot simpler, and is suitable for enhancing static HTML or integrating with a backend framework. However, you won't be able to use the Single-File Component (SFC) syntax.`

There is a project which does this, but I would love to see this feature get officially adopted.
https://github.com/FranckFreiburger/vue3-sfc-loader

Please support loading "Single File Components" (.vue files) dynamically at runtime, even when Vue is loaded from a CDN.

**Comment by nickmessing** [comment]
That sounds expensive. The library in question already is almost 3MB big, and the use-case is really extreme. Using everything at runtime is both CPU and network-intensive and spending developer effort to have one of the worst possible patterns maintained does not really seem like a good idea.
